manifests/init.pp

```diff
@@ -123,7 +123,9 @@
   validate_re($service_ensure, '^running|true|stopped|false$')
 
   anchor { 'sssd::begin': } ->
-  class { '::sssd::install': } ->
+  class { '::sssd::install':
+    ensure => $sssd_package_ensure,
```

Comment:
I think that reading this var from the install class is cleaner...
Can we leave this code alone?  Is there a reason to change it?